client/state/action-types.ts

```diff
@@ -619,6 +619,9 @@ export const PLUGINS_RECEIVE = 'PLUGINS_RECEIVE';
 export const PLUGINS_REQUEST = 'PLUGINS_REQUEST';
 export const PLUGINS_REQUEST_FAILURE = 'PLUGINS_REQUEST_FAILURE';
 export const PLUGINS_REQUEST_SUCCESS = 'PLUGINS_REQUEST_SUCCESS';
+export const ALL_PLUGINS_REQUEST = 'ALL_PLUGINS_REQUEST';
```

Comment:
We usually try to keep this list alphabetized. Also, I'd suggest keeping the `PLUGINS_` prefix here. `PLUGINS_ALL_*` sounds okay.

Reply:
Thanks! I agree `PLUGINS_ALL_*` sounds better and I've updated this in 839e7834a1f19087233fee5137dc89ec9bb1a5e2